src/borg/testsuite/helpers.py

```diff
@@ -1041,3 +1043,21 @@ def test_replace_placeholders():
     now = datetime.now()
     assert " " not in replace_placeholders('{now}')
     assert int(replace_placeholders('{now:%Y}')) == now.year
+
+
+def working_swidth():
+    return platform.swidth('선') > 1
```

Comment:
all these strings i can't read, do you know what they mean and did you avoid offensive or otherwise questionable stuff?